CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Unreleased
 
+- Minor: Add advanced setting to hide chat when screenshotting. (#208)
```

Comment:
```suggestion
- Minor: Add advanced setting to hide chat (and PMs above chat) when screenshotting. (#208)
```

Reply:
meh - if they don't want PMs hidden also, it's just as simple as hiding the widget manually, and not using the setting.